gocov-xml.go

```diff
@@ -80,7 +93,7 @@ func main() {
 			className, methodName := s[len(s)-2], s[len(s)-1]
 			class := classes[className]
 			if class == nil {
-				class = &Class{Name: className, Filename: gFunction.File, Methods: []Method{}, Lines: []Line{}}
+				class = &Class{Name: className, Filename: gFunction.File, Methods: []Method{}, Lines: []Line{}, LineCount: 0, LineHits: 0}
```

Comment:
Go always initializes everything with zero values, no need to be explicit about it.

Reply:
not find a way for how to avoid 0 in structs field:value init. 
Sorry not too much experience on Go...